sw/airborne/arch/stm32/px4fmu_2.4.ld

```diff
@@ -19,15 +19,14 @@
  * Boston, MA 02111-1307, USA.
  */
 
-/* Linker script for Pixhawk PX4FMU v2.4 (STM32F427, 2048K flash, 256K RAM). */
+/* Linker script for Pixhawk PX4FMU v2.4 (STM32F427, 1024K flash, 192K RAM). */
 
 /* Define memory regions. */
 MEMORY
 {
-	/* only 192K (SRAM1 and SRAM2) are accessible by all AHB masters. */
-	ram (rwx) : ORIGIN = 0x20000000, LENGTH = 192K
+    ram (rwx) : ORIGIN = 0x20000000, LENGTH = 192K
     /* Reserving 128kb flash for persistent settings. */
-    rom (rx) : ORIGIN = 0x08000000, LENGTH = 1920K
+    rom (rx) : ORIGIN = 0x08004000, LENGTH = 1008K
```

Comment:
I got that from the PX4 repo [ld script](https://github.com/PX4/Firmware/blob/afb68cb0abee4034582f93d7f383c287b9c9a109/nuttx-configs/px4fmu-v2/scripts/ld.script). See the comments over there. Particularly note the "Disabled due to silicon errata flash (rx)". They have had a badge of Pixhawks with bad flash...